src/screens/Settings/index.tsx

```diff
@@ -25,10 +28,32 @@ export function SettingsList() {
     const handlePlaybackReportingClick = useCallback(() => { navigation.navigate('Playback Reporting'); }, [navigation]);
     const handleColorSchemeClick = useCallback(() => { navigation.navigate('Color Scheme'); }, [navigation]);
     const handlePrivacyPolicyClick = useCallback(() => { navigation.navigate('PrivacyPolicy'); }, [navigation]);
+    const handleTimerClick = useCallback(() => { navigation.navigate('Timer'); }, [navigation]);
+
+    const { sleepTime } = useTypedSelector(state => state.settings);
+
+    const getTime = () => {
+        let hours = 0;
+        let minutes = 0;
+        if (!Number.isNaN(sleepTime)) {
+            hours = Math.round(sleepTime / 3600);
+            const timeRemaining = sleepTime % 3600;
+
+            if (timeRemaining >= 60) {
+                minutes = Math.round(timeRemaining / 60);
+            }
+        }
+
+        return `${hours} hrs ${minutes} min`;
+    }
```

Comment:
It's unnecessary to show the set timer in this view. Also, if you're doing calculations based on another value, please memoize it using `useMemo`, rather than putting it in an arrow function.